client/express-checkout/blocks/index.js

```diff
@@ -16,7 +16,17 @@ const expressCheckoutElementPaymentMethod = ( api ) => ( {
 	supports: {
 		features: getConfig( 'features' ),
 	},
-	canMakePayment: () => wcpayConfig.isExpressCheckoutElementEnabled,
+	canMakePayment: () => {
+		if ( typeof wcpayExpressCheckoutParams === 'undefined' ) {
```

Comment:
Should `wcpayExpressCheckoutParams` also be imported in this file?
```javascript
/* global wcpayConfig, wcpayExpressCheckoutParams */

```

Reply:
It’s not really necessary, but I added it as a precaution. https://github.com/Automattic/woocommerce-payments/pull/8854/commits/7c31da257e9626f36a4962ab9acd5c330c2096d6